tools/ratbagctl.test.in

```diff
@@ -374,6 +374,7 @@ class TestRatbagCtlDPI(TestRatbagCtl):
         { "is_active": false,
           "resolutions": [
             { "xres": 1000, "yres": 1100,
+              "capabilities": [1],
```

Comment:
Since it's not failing that's already a good indication, but if you look at the code (line 407), it's expected to have `1300x1400dpi`, while in fact that should result in an error. The problem with the test code right now is that no capabilities for the resolutions are actually checked. At some point I may add this, since it's useful with the disabled capability.